views/author_delete.pug

```diff
@@ -8,22 +8,19 @@ block content
   if author_books.length
 
     p #[strong Delete the following books before attempting to delete this author.]
-
     div(style='margin-left:20px;margin-top:20px')
-
       h4 Books
-
       dl
-      each book in author_books
-        dt
-          a(href=book.url) #{book.title}
-        dd #{book.summary}
+        each book in author_books
+          dt
+            a(href=book.url) #{book.title}
+          dd #{book.summary}
 
   else
     p Do you really want to delete this Author?
 
-    form(method='POST' action='')
+    form(method='POST')
```

Comment:
This one _might_ be needed - we will need to check the docs.

To be more precise, if the action is not specified then I believe that the post is just back to the same URL. The checker will be suggesting it is removed because it is not necessary to specify at all if you are not using it. BUT the tutorial might well do this deliberately, so we should co-ordinate a change.

Reply:
The action attribute in a form must not be empty. If it is missing the form is send to the original url.
https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fs-action